src/atomate2/vasp/flows/core.py

```diff
@@ -42,12 +42,16 @@ class DoubleRelaxMaker(Maker):
     ----------
     name : str
         Name of the flows produced by this maker.
-    relax_maker : .BaseVaspMaker
-        Maker to use to generate the relaxations.
+    relax_maker1 : .BaseVaspMaker
+        Maker to use to generate the first relaxation.
+    relax_maker2 : .BaseVaspMaker
+        Maker to use to generate the second relaxation. If not set,
+        then relax_maker2 = relax_maker1.
     """
 
     name: str = "double relax"
-    relax_maker: BaseVaspMaker = field(default_factory=RelaxMaker)
+    relax_maker1: BaseVaspMaker = field(default_factory=RelaxMaker)
+    relax_maker2: BaseVaspMaker = relax_maker1
```

Comment:
I'm confused that this works? As I thought relax_maker1 is a class variable of the Maker and not available in global scope. Although maybe this is a data class trick?

At the very least, does this not mean that relax_maker1 and relax_maker2 are the same object. So if you update one then you update the other?

I think the two options would be:
1. Make a deepcopy of the relax_maker1.
2. Set relax_maker2 to None and then use relax_maker1 if relax_maker2 is not set.

Reply:
When I try just a simple test case, this gives weird results:

```python
from dataclasses import dataclass, field

@dataclass
class Test:
    a: list = field(default_factory=lambda: [1, 2, 3])
    b: list = a
    
Test().a
# Field(name='b',type=<class 'list'>,default=<dataclasses._MISSING_TYPE object at 0x1037cd4f0>,
#       default_factory=<function Test.<lambda> at 0x15a28c310>,init=True,repr=True,hash=None,
#       compare=True,metadata=mappingproxy({}),_field_type=_FIELD)
```

Do the tests definitely work on your system?

Reply:
Having thought about this a bit more. I think the best behaviour would be just to have two separate makers. I.e., if you update the first relax maker that will have no effect on the second relax maker. This is how nested makers work in all other Flows so probably best to be consistent here.

We could add a utility class method to DoubleRelaxMaker called `from_relax_maker(relax_maker)`. This would allow you to initialise a double relaxation with the same relax_maker used for both relaxations. I think then most of the existing workflows that use the double relax maker should then use this utility function.

Reply:
> When I try just a simple test case, this gives weird results:
> 
> ```python
> from dataclasses import dataclass, field
> 
> @dataclass
> class Test:
>     a: list = field(default_factory=lambda: [1, 2, 3])
>     b: list = a
>     
> Test().a
> # Field(name='b',type=<class 'list'>,default=<dataclasses._MISSING_TYPE object at 0x1037cd4f0>,
> #       default_factory=<function Test.<lambda> at 0x15a28c310>,init=True,repr=True,hash=None,
> #       compare=True,metadata=mappingproxy({}),_field_type=_FIELD)
> ```
> 
> Do the tests definitely work on your system?

Sorry for going AWOL on this one. I had trouble getting the test suite working (have to go back and fix that) and was relying on the CI for what I thought would be a quick fix. Turns out I was a bit off the mark here (only started writing my own `Maker`very recently). Apologies for the lack of clarity.